action.yml

```diff
@@ -25,7 +25,7 @@ inputs:
   use_osv_scanner:
     description: "Use Datadog osv-scanner to produce an SBOM"
     required: false
-    default: "false"
+    default: "true"
```

Comment:
Can you update the readme?

Reply:
What part of the `README` do you want me to update?

Reply:
Oh I thought we had an inputs similar to this: https://github.com/DataDog/datadog-static-analyzer-github-action?tab=readme-ov-file#inputs